<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CH9326Dlg</class>
 <widget class="QWidget" name="CH9326Dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CH9326DEBUG</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>561</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string>   Device Conn</string>
   </property>
   <widget class="QLineEdit" name="m_deviceInfo">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Device</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_openDevice">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QComboBox" name="m_deviceName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Device Info</string>
    </property>
   </widget>
   <widget class="QComboBox" name="m_baudCb">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>97</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>300</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>28800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>76800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="BaudRateLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>105</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Baud rate:</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="buddy">
     <cstring>m_baudCb</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="ParityLabel">
    <property name="geometry">
     <rect>
      <x>268</x>
      <y>104</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Parity :</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="buddy">
     <cstring>m_parityCb</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="m_parityCb">
    <property name="geometry">
     <rect>
      <x>344</x>
      <y>97</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Odd</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Even</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Space</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="m_setConfig">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>97</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>SetCfg</string>
    </property>
   </widget>
   <widget class="QLabel" name="StopBitsLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>141</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop bits:</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="buddy">
     <cstring>m_stopBitsCb</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="m_stopBitsCb">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>135</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="DataBitsLabel">
    <property name="geometry">
     <rect>
      <x>268</x>
      <y>136</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Data bits:</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="buddy">
     <cstring>m_dataBitsCb</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="m_dataBitsCb">
    <property name="geometry">
     <rect>
      <x>344</x>
      <y>133</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="m_sendArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>197</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="BaudRateLabel_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>175</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Send Data(Number &lt; 32)</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="buddy">
     <cstring>m_baudCb</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="BaudRateLabel_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>266</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Receive Data</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="buddy">
     <cstring>m_baudCb</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_edit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>266</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_sendBtn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>199</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_rcvBtn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>297</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Receive</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_rcvStopBtn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>342</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Receive</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_rcvClearBtn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>387</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>ClearData</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="m_recvArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>401</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_hexInputCb">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>267</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>HEX Input</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_hexOutputCb">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>HEX Output</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="m_groupBox1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>450</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string> IO1</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QRadioButton" name="m_inBtn1">
      <property name="text">
       <string>IO1 IN</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">BtnGroup11</string>
      </attribute>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="m_highBtn1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>IO1 High</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">BtnGroup12</string>
      </attribute>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="m_outBtn1">
      <property name="text">
       <string>IO1 OUT</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">BtnGroup11</string>
      </attribute>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QRadioButton" name="m_lowBtn1">
      <property name="text">
       <string>IO1 Low</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">BtnGroup12</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="m_setIoDirBtn">
   <property name="geometry">
    <rect>
     <x>67</x>
     <y>616</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Set IoDir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_outputBtn">
   <property name="geometry">
    <rect>
     <x>242</x>
     <y>616</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Output IO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_inputBtn">
   <property name="geometry">
    <rect>
     <x>417</x>
     <y>616</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Input IO</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="m_groupBox2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>450</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string> IO2</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QRadioButton" name="m_inBtn2">
      <property name="text">
       <string>IO2 IN</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">BtnGroup21</string>
      </attribute>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="m_highBtn2">
      <property name="text">
       <string>IO2 High</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">BtnGroup22</string>
      </attribute>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="m_outBtn2">
      <property name="text">
       <string>IO2 OUT</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">BtnGroup21</string>
      </attribute>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QRadioButton" name="m_lowBtn2">
      <property name="text">
       <string>IO2 Low</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">BtnGroup22</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="m_groupBox3">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>528</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string> IO3</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QRadioButton" name="m_inBtn3">
      <property name="text">
       <string>IO3 IN</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">BtnGroup31</string>
      </attribute>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="m_highBtn3">
      <property name="text">
       <string>IO3 High</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">BtnGroup32</string>
      </attribute>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="m_outBtn3">
      <property name="text">
       <string>IO3 OUT</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">BtnGroup31</string>
      </attribute>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QRadioButton" name="m_lowBtn3">
      <property name="text">
       <string>IO3 Low</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">BtnGroup32</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="m_groupBox4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>528</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string> IO4</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QRadioButton" name="m_inBtn4">
      <property name="text">
       <string>IO4 IN</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">BtnGroup41</string>
      </attribute>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="m_highBtn4">
      <property name="text">
       <string>IO4 High</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">BtnGroup42</string>
      </attribute>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="m_outBtn4">
      <property name="text">
       <string>IO4 OUT</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">BtnGroup41</string>
      </attribute>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QRadioButton" name="m_lowBtn4">
      <property name="text">
       <string>IO4 Low</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">BtnGroup42</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="BtnGroup32"/>
  <buttongroup name="BtnGroup21"/>
  <buttongroup name="BtnGroup22"/>
  <buttongroup name="BtnGroup11"/>
  <buttongroup name="BtnGroup12">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
  <buttongroup name="BtnGroup41"/>
  <buttongroup name="BtnGroup42"/>
  <buttongroup name="BtnGroup31"/>
 </buttongroups>
</ui>
